Sources/BraveWallet/Crypto/NetworkPicker.swift

```diff
@@ -59,11 +62,19 @@ struct NetworkPicker: View {
   
   var body: some View {
     Button(action: {
-      networkSelectionStore = .init(networkStore: networkStore)
+      networkSelectionStore = .init(
+        mode: .select(isForOrigin: isForOrigin),
+        networkStore: networkStore
+      )
     }) {
       HStack {
-        Text(networkStore.selectedChain.chainName)
-          .fontWeight(.bold)
+        if isForOrigin {
+          Text(networkStore.selectedChainForOrigin.chainName)
+            .fontWeight(.bold)
+        } else {
+          Text(networkStore.defaultSelectedChain.chainName)
+            .fontWeight(.bold)
+        }
```

Comment:
do 
Text(isForOrigin ? networkStore.selectedChainForOrigin.chainName : networkStore.defaultSelectedChain.chainName)
.fontWeight(.bold)
instead?

Reply:
Updated in [084d05c](https://github.com/brave/brave-ios/pull/7470/commits/084d05c615b0b0bf277b3bb1132d0202a73febcc). Moved to computed property to keep line length down.